spec/rubocop/ast/send_node_spec.rb

```diff
@@ -1153,6 +1153,13 @@ module Foo
       it { expect(send_node.lambda?).to be_truthy }
     end
 
+    context 'with a method named lambda in a class' do
+      let(:source) { 'foo.lambda { |bar| baz }' }
+      let(:send_node) { parse_source(source).ast.send_node }
+
+      it { expect(send_node.lambda?).to be_falsey }
```

Comment:
This is a small point in the wonderful PR as a whole 😃
Since `be_falsey` treats `nil` as true, I think that `be false` is preferable for behaviors that return boolean value.

Reply:
This sounds like it could be a cop for `rubocop-rspec`. WDYT @koic? 🙂

Reply:
Sure. I thinks so too!

Reply:
Will keep this on par with the rest of the specs for now, then. We can change them all using auto-correct when we implement the cop in `rubocop-rspec`. 🙂

Reply:
Yep. I think that's fine 😃

Reply:
//cc @rubocop-hq/rubocop-rspec :-)

Reply:
> for behaviors that return boolean value

Therein lies the challenge. We could look at whether the method called ends with `?`, but other than that I think we’d need dynamic analysis.

Perhaps RSpec should emit a warning when `be_falsey` matches a literal false, and `be_truthy` matches a literal true?

Reply:
Exactly, I thought of the cop targeting method names that end with `?`.

On the other hand, it may be necessary to judge carefully that RSpec dynamically gives warnings. For example, a large amount of logs may bother users during spec runs 🤔